operator/v1/types_network.go

```diff
@@ -137,6 +137,15 @@ type NetworkMigration struct {
 	// supported features.
 	// +optional
 	Features *FeaturesMigration `json:"features,omitempty"`
+
```

Comment:
You're right. Users add this struct. CNO deploys the new CNI plugin, it will report the status in conditions of the network cluster operator CR. Users remove this struct once complete. CNO remove the old CNI plugin.